notebooks/processing/PPE_needs_for_confirmed_covid-19_at_county_level.ipynb

```diff
@@ -0,0 +1,2973 @@
+    "As the number of covid cases in the states continues to grow, U.S. hospitals are already reporting shortages of key equipment needed to care for critically ill patients, including ventilators and personal protective equipment (PPE) for medical staff. \n",
+    "\n",
+    "![convid19_patient_flow](https://user-images.githubusercontent.com/14057932/77640828-695cee00-6f31-11ea-812b-55c33f2e9de4.png)\n",
+    "\n",
+    "*Fig 1* PPE needs through confirmed convid19 case. \n",
+    "\n",
+    "\n",
+    "### PPEs in this case include:\n",
+    "- Convid19 virus testkit;\n",
+    "- Droplet mask/surgical mask;\n",
+    "- N95 mask\n",
+    "- Gloves\n",
+    "- Gown\n",
+    "- Faceshields\n",
+    "- Googles\n",
+    "- BP cuff, stethoscope, pulse Ox, Ambu bug\n",
+    "- Sani-wipers\n",
+    "- Ventilators\n",
+    "\n",
+    "\n",
+    "## Input\n",
+    "\n",
+    "**The input data is the confirmed covid19 cases at county level** (Data from [USAFacts](https://usafacts.org/visualizations/coronavirus-covid-19-spread-map/), CSV data can be found [here](../../data/external/covid_confirmed_usafacts.csv)). We took in the newly added case daily for the PPE needs estimation. \n",
+    "\n",
+    "The PPEs are consumed through the covid19 patient flow that has shown in the above diagram. \n",
+    "\n",
+    "In order to make the estimation, we assumed:\n",
+    "\n",
+    "- 14% test cases are tested as positive. This can change from county to county. \n",
+    "- 80% of the confirmed positive cases will have very mild symptoms and only need home care; while\n",
+    "- 20% of the confirmed cases will end up in in-patient care; \n",
+    "- 40% of patients from in-patient care may need critical care. \n",
+    "\n",
+    "- A patient may have about ten-room visits from doctors and nurses per day in in-patient care; but \n",
+    "- A patient in critical care may need a room visit every 20 - 30 mins from doctors, nurses, other room visits may come from therapists, ventilator technicians. We assume 40 room visits per day for a critical care patient.\n",
+    "\n",
+    "- Each patient need a ventilator in critical care;\n",
+    "\n",
+    "- A patient may spend seven days in in-patient care;\n",
+    "- A patient may spend an average of 10 days at critical care. \n",
+    "\n",
+    "\n",
+    "\n",
+    "## Output\n",
+    "\n",
+    "**County-level total PPE needs estimation from the current confirmed case from Jan 22nd to March 24th, 2020**"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import os.path as op\n",
+    "from IPython.display import Image\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import geopandas as gpd"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cases_file = \"../../data/external/covid_confirmed_usafacts.csv\"\n",
```

Comment:
Can you name this file to the specific date it has, or the `Last Update` column value? That way people know it's a static file, and don't confuse it with the live-updating data source.

Also, making a note in about what the data and where it came from in https://github.com/covidcaremap/covid19-healthsystemcapacity/blob/master/data/README.md#external-datasets will help us keep track of source data.

Reply:
Thanks for reviewing my PR @lossyrob, I will edit them accordingly.